internal/attestation/snp/validator.go

```diff
@@ -58,10 +58,13 @@ func (v *Validator) OID() asn1.ObjectIdentifier {
 func (v *Validator) Validate(attDocRaw []byte, nonce []byte, peerPublicKey []byte) (err error) {
 	v.logger.Info("Validate called", "nonce", hex.EncodeToString(nonce))
 	defer func() {
+		// Note: We tolerate Validate() to fail for validators without matching configuration. Only 1 of n validators has to succeed.
+		// Thus the error handling of the validators takes place in the calling function verifyEmbeddedReport().
+		// Do not log any errors in the subroutines and use info to still provide insights of the validation processes.
 		if err != nil {
-			v.logger.Error("Validation failed", "error", err)
+			v.logger.Info("Validation failed", "nonce", hex.EncodeToString(nonce), "error", err)
 		} else {
-			v.logger.Info("Validation successful")
+			v.logger.Info("Validation successful", "nonce", hex.EncodeToString(nonce))
```

Comment:
Do we need to log the nonce? If validation fails because of it, wouldn't we see that in the corresponding error?

Reply:
The current error message didn't contain nonce logging. I added logging the nonce as identifier to the atls connection and the oid as identifier of the validator into the our current logging. 
On log-level DEBUG, the single validation failures are logged as well. Otherwise just the successful validation call is logged with the corresponding nonce and OID.